lib/server.js

```diff
@@ -314,12 +313,12 @@ Server.prototype.handshake = function(transportName, req){
  * @api public
  */
 
-Server.prototype.handleUpgrade = function(req, socket, upgradeHead){
+Server.prototype.handleUpgrade = function (req, socket, upgradeHead) {
   this.prepare(req);
 
   var self = this;
-  this.verify(req, true, function(err, success) {
-    if (!success) {
+  this.verify(req, true, function (err, success) {
+    if (err) {
```

Comment:
ESLint requires us to make use of `err` when handling the error callback. Should I make an exception to the rule here, or is the way I handle it currently is fine? It passes the test now.